src/framework/Framework_DeviceInfo_001.test.js

```diff
@@ -0,0 +1,72 @@
+import {
+  getPluginInfo,
+  pluginDeactivate,
+  pluginActivate,
+  restartFramework,
+} from '../commonMethods/commonFunctions'
+import constants from '../commonMethods/constants'
+import { restartFramework1 } from '../../../../src/commonFunctions1'
+
+export default {
+  title: 'Framework Device Info test',
+  description: 'Validates functionality of the Device Info plugin',
+  teardown: restartFramework1,
```

Comment:
restartFramework1 looks like a rollup error? :) anyway teardown can be removed.

Reply:
Remove tearDown method.